bin/dockerd-wrapper

```diff
@@ -48,4 +57,6 @@ set -- \
 
 workaround_lp1626019
 
-exec docker daemon "$@"
+workaround_lp1606510
+
+exec dockerd -G $default_socket_group "$@"
```

Comment:
I've double-checked here.
If -G is specified with "docker" group and the group doesn't exist on OS.
Error popups as docker.sock is owned by host root, with docker group ownership.
http://paste.ubuntu.com/24582063/
Basically, that's the same behavior if no -G option specified for daemon startup. As docker group is the default group for the unix socket.

So a user has to 1) add sudo to make docker command works. 2) create a docker group and add the login user into it so that they can run docker command without sudo.

On Ubuntu Core16, docker group is pre-shipped.